polynote-frontend/polynote/ui/layout/splitview.ts

```diff
@@ -9,12 +9,29 @@ import {ViewPreferences, ViewPrefsHandler} from "../../state/preferences";
 export type Pane = { header: TagElement<"h2">, el: TagElement<"div">}
 export class SplitView extends Disposable {
     readonly el: TagElement<"div">;
+    private _centerWidth: number = 0;
+    get centerWidth(): number { return this._centerWidth; }
+
+    private observers: (((width: number) => void) & IDisposable)[] = [];
+
     constructor(leftPane: Pane, center: TagElement<"div">, rightPane: Pane) {
         super()
 
         const leftView = ViewPrefsHandler.lens("leftPane").disposeWith(this);
         const rightView = ViewPrefsHandler.lens("rightPane").disposeWith(this);
-        const triggerResize = () => window.dispatchEvent(new CustomEvent('resize'));
+        const triggerResize = () => {
```

Comment:
is `triggerResize` referenced anywhere? looks like all the references are for `this.triggerResize` unless I'm missing something.

Reply:
There are a couple of calls to `triggerResize`. It happens when a panel is collapsed or un-collapsed. I'll untangle this because it's really bad and confusing.